hw3/src/pages/home.jsx

```diff
@@ -12,6 +12,12 @@ const Header = () => {
         South American countries. Navigate on our website to see list of
         contries in South America, their population, flags and more!
       </p>
+      <img
+        src="../public/South_America.svg.png"
+        alt="South America Map"
+        className="mt-8 mx-auto max-w-full"
+        style={{ maxWidth: "250px" }}
+      />
```

Comment:
This might just be an image of the continent. However, if used with other continents, it gives the person an aspect of where the continent is.